diff --git a/bin/review-catalog-converter b/bin/review-catalog-converter
index ce4bc65..2715569 100755
--- a/bin/review-catalog-converter
+++ b/bin/review-catalog-converter
@@ -13,7 +13,7 @@
 require 'pathname'
 require 'optparse'
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 require 'review/version'
 require 'review/extentions'
 
diff --git a/bin/review-check b/bin/review-check
index d5a61ee..45592e7 100755
--- a/bin/review-check
+++ b/bin/review-check
@@ -13,7 +13,7 @@
 require 'pathname'
 
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 
 require 'review'
 require 'optparse'
diff --git a/bin/review-checkdep b/bin/review-checkdep
index d15246d..24c6137 100755
--- a/bin/review-checkdep
+++ b/bin/review-checkdep
@@ -13,7 +13,7 @@
 require 'pathname'
 
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 
 PREDEF_FILE = 'PREDEF'
 
diff --git a/bin/review-compile b/bin/review-compile
index 547907f..60d3022 100755
--- a/bin/review-compile
+++ b/bin/review-compile
@@ -14,7 +14,7 @@
 require 'pathname'
 
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 
 require 'review'
 require 'fileutils'
diff --git a/bin/review-epubmaker b/bin/review-epubmaker
index fe97e0a..cbc07ce 100755
--- a/bin/review-epubmaker
+++ b/bin/review-epubmaker
@@ -11,7 +11,7 @@
 require 'pathname'
 require 'optparse'
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 require 'review/epubmaker'
 require 'review/version'
 
diff --git a/bin/review-index b/bin/review-index
index 014d12b..acad453 100755
--- a/bin/review-index
+++ b/bin/review-index
@@ -13,7 +13,7 @@
 require 'pathname'
 
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 
 require 'review/book'
 require 'review/tocparser'
diff --git a/bin/review-init b/bin/review-init
index e88e769..bfd8f0c 100755
--- a/bin/review-init
+++ b/bin/review-init
@@ -13,7 +13,7 @@ require 'optparse'
 require 'pathname'
 
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 
 require 'review/version'
 
diff --git a/bin/review-pdfmaker b/bin/review-pdfmaker
index ef7ae57..bafa9ba 100755
--- a/bin/review-pdfmaker
+++ b/bin/review-pdfmaker
@@ -11,7 +11,7 @@
 
 require 'pathname'
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 
 require 'review/pdfmaker'
 
diff --git a/bin/review-preproc b/bin/review-preproc
index 7546da9..cb6fae3 100755
--- a/bin/review-preproc
+++ b/bin/review-preproc
@@ -14,7 +14,7 @@
 require 'pathname'
 
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 
 require 'review/preprocessor'
 require 'review/unfold'
diff --git a/bin/review-vol b/bin/review-vol
index f7ad145..37f229b 100755
--- a/bin/review-vol
+++ b/bin/review-vol
@@ -13,7 +13,7 @@
 require 'pathname'
 
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 
 require 'review'
 require 'optparse'
diff --git a/bin/review-webmaker b/bin/review-webmaker
index 18d9f38..4671452 100755
--- a/bin/review-webmaker
+++ b/bin/review-webmaker
@@ -8,7 +8,7 @@
 
 require 'pathname'
 bindir = Pathname.new(__FILE__).realpath.dirname
-$LOAD_PATH.unshift((bindir + '../lib').realpath)
+$LOAD_PATH.unshift("/usr/share/review")
 
 require 'review/webmaker'
 
-- 
2.15.0

diff --git a/lib/review/template.rb b/lib/review/template.rb
index f311c74..9fb8600 100644
--- a/lib/review/template.rb
+++ b/lib/review/template.rb
@@ -4,7 +4,7 @@ module ReVIEW
   class Template
     include ERB::Util
 
-    TEMPLATE_DIR = File.join(File.dirname(__FILE__), "../../templates")
+    TEMPLATE_DIR = File.join(File.dirname(__FILE__), "../templates")
 
     def self.load(filename, mode = 1)
       self.new(filename, mode)
